EnglishFilter/sections/foreign.txt

```diff
@@ -3160,6 +3160,8 @@ sentireascoltare.com##.banner_scroll_space
 @@||justnerd.it^$generichide
 !+ PLATFORM(ext_ff, ext_opera, ios, ext_android_cb, ext_ublock)
 justnerd.it##.adsbygoogle
+! https://github.com/AdguardTeam/AdguardFilters/issues/38417
+@@||vendorlist.consensu.org/vendorlist.json$domain=cmp.fanpage.it
```

Comment:
It's blocked by `AdGuard Annoyances filter`, so please move this rule to `cookies_whitelist.txt`